android/build.gradle

```diff
@@ -1,12 +1,12 @@
 buildscript {
-    ext.kotlin_version = '1.2.71'
+    ext.kotlin_version = '1.3.50'
```

Comment:
this particular upgrade is not necessary for this PR, but is done anyways to keep up to date with master flutter.